jmix-rest/rest/src/main/java/io/jmix/rest/RestProperties.java

```diff
@@ -38,15 +39,22 @@ public class RestProperties {
     private final int defaultMaxFetchSize;
     private final Map<String, Integer> entityMaxFetchSize;
 
+    /**
+     * File extensions that can be opened for viewing in a browser.
```

Comment:
Add more details:
"File extensions that can be opened for viewing in a browser by replying with 'Content-Disposition=inline' header"

Reply:
Done